### ISSUE TITLE
Update e2e testing to focus on percy snapshot testing for Web Components

### ISSUE THREAD

**Issue by jeffchew** [carbon-design-system/carbon-for-ibm-dotcom#5612]
#### User Story

> As a `[user role below]`:
Carbon for IBM.com tester

> I need to:
see automated visual regression tests against knob variations of storybook stories in Web Components

> so that I can:
properly test additional states of components in Browserstack/Percy

#### Additional information

 - Previous setup has been done for React: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/5474
 - Some stubs were made with Web Components, but snapshots are not rendering properly for some reason

#### Acceptance criteria
 - [ ] Updated harness to have working snapshots of percy tests in Web Components
 - [ ] Hook up CI pipeline for e2e tests to generate for Web Components

**Comment by jeffchew** [comment]
Taking a look at this, and discovering that this may be a limitation on how the DOM is captured in Percy, serialized, and sent to the Percy build. I added a comment to the following existing issue found under `@percy/dom` within their CLI:

https://github.com/percy/cli/issues/280

The only workaround I found was to set `enableJavascript` to `true`, which does seem to render the shadow DOM content. However, it does not seem to capture any interactive states on the page. This includes clicking menus, entering in search forms, etc. 

We could feasibly focus on non-interactive snapshots like theming, knob settings, etc. Or we can shelve this entirely until the feature is available from Percy.
